### DISCUSSION TITLE
Porkbun API Endpoint Change

### DISCUSSION THREAD

**Discussion by nathanielhourt** [go-acme/lego#2303] [Q&A] [answered]
This is a question regarding the upcoming Porkbun API endpoint change. Lego supports Porkbun as a DNS provider, and they are moving their API URL from `porkbun.com/api` to `api.porkbun.com/api`.

Lego [seems to use](https://github.com/go-acme/lego/blob/master/providers/dns/porkbun/porkbun.go#L15) nrdcg/porkbun to call the API, and that repo adopted the new URL [back in June](https://github.com/nrdcg/porkbun/commit/b87518cc5512a15751f6834dd700a2555119f9f5)... So is that all settled, then? No further action is necessary, and it will continue working?

I just wanted to make sure and I haven't found any discussion on it anywhere. =) TIA

**Comment by ldez** [comment, marked as answer]
The API URL has been updated there are 4 months inside the API client: https://github.com/nrdcg/porkbun/pull/7

The API client has been updated inside this PR: https://github.com/go-acme/lego/pull/2272

And this PR is a part of v4.19.0

https://github.com/go-acme/lego/pull/2299#issuecomment-2408241087
